### ISSUE TITLE
how to download all video from a user

### ISSUE THREAD

**Issue by sadazs** [XavierJiezou/python-dy-down#1]
Hi Xavi, do you know how to download all video from an user, when copy an user's link it display 50 newest video not all so can't grab all video link

**Comment by XavierJiezou** [comment]
Yes, I have a solution. You can view my csdn blog: [https://blog.csdn.net/qq_42951560/article/details/111227342](https://blog.csdn.net/qq_42951560/article/details/111227342)
